examples/mood.py

```diff
@@ -0,0 +1,43 @@
+"""MooD - a pacifist/tourist meta-mod for Doom
+
+All monsters are pacifist and will not attack the player or each other.
+The pistol has been replaced by the fist.
+All weapon pickups have been replaced by the Health Potion.
+
+For more information: https://jmtd.net/doom/mood/"""
+
+from deh9000 import *
+
+# pacifist monsters
+for monster in mobjinfo:
+    if not (monster.flags & MF_COUNTKILL or monster == mobjinfo[MT_SKULL]):
+        continue
+
+    if monster.missilestate:
+        states[monster.missilestate].tics = 1
```

Comment:
Is it sufficient to just do:

    monster.missilestate = monster.seestate

?

Reply:
sadly not, chocolate segfaults when the event occurs (I mean to separately file/fix that in chocolate)

Reply:
however there might be other, nicer approaches to achieve something similar. I was saving them for a future hack session.